sql/src/test/java/io/druid/sql/avatica/DruidAvaticaHandlerTest.java

```diff
@@ -96,7 +96,7 @@
     @Override
     public int getMaxConnections()
     {
-      return 2;
+      return 3;
```

Comment:
Could you leave a comment, or extract a constant, or somehow else prevent regression in the future?

Reply:
added a comment